pkg/template/versiontemplate.go

```diff
@@ -24,7 +24,7 @@ func VersionTemplate(test VersionTestTemplate) {
 		}
 	}
 
-	err := checkVersion(test)
+	err := executeTestCombination(test)
```

Comment:
every call of this func is refactored right ?

Reply:
Fixed